README.md

```diff
@@ -30,4 +52,37 @@ yield scrapy.Request("http://books.toscrape.com/",
                        "echoData": {"something": True}
                    }
                }),
+```
+
+## Example Code:
+
+```python
+import scrapy
+import os
+
+
+class TestSpider(scrapy.Spider):
+    name = 'test'
+    os.environ["ZYTE_API_KEY"] = "<You ZYTE_API_KEY>"
```

Comment:
This wouldn't be a standard way of defining things in Scrapy. Thanks fo raising this!

I've opened up an issue in https://github.com/scrapy-plugins/scrapy-zyte-api/issues/4 so it can be set via the `settings.py` instead. I'll work on it soon.

Reply:
Cool, that would be great, yea this becomes a problem when we have to use windows, and setting an env variable on windows is a kind of a headache.

So for now, should I keep it for documentation for now until it is implemented to mention it in `settings.py`?

Reply:
I think we can remove this one as the upcoming PR in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/5 mentions this in the documentation.

Reply:
Yes I removed this part now.